iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/SubstationXml.java

```diff
@@ -44,11 +46,17 @@ protected void writeRootElementAttributes(Substation s, Network n, NetworkXmlWri
         if (s.getTso() != null) {
             context.getWriter().writeAttribute("tso", context.getAnonymizer().anonymizeString(s.getTso()));
         }
-        if (!s.getGeographicalTags().isEmpty()) {
-            context.getWriter().writeAttribute("geographicalTags", s.getGeographicalTags().stream()
-                    .map(tag -> context.getAnonymizer().anonymizeString(tag))
-                    .collect(Collectors.joining(",")));
-        }
+        IidmXmlUtil.runUntilMaximumVersion(IidmXmlVersion.V_1_2, context, () -> {
+            if (!s.getGeographicalTags().isEmpty()) {
+                try {
+                    context.getWriter().writeAttribute(Substation.GEOGRAPHICAL_TAGS_KEY, s.getGeographicalTags().stream()
+                            .map(tag -> context.getAnonymizer().anonymizeString(tag))
+                            .collect(Collectors.joining(",")));
+                } catch (XMLStreamException e) {
+                    throw new UncheckedXmlStreamException(e);
+                }
+            }
+        });
```

Comment:
Don't use that, simply delete this part: the geographical tags will be written as properties in any case if they are defined. However, you will need to change the read method: until 1.2, you can read geographicalTags and set them as properties.